components/prize_pool/commons/prize_pool.lua

```diff
@@ -198,9 +210,80 @@ function PrizePool:create()
 end
 
 function PrizePool:build()
-	-- TODO
-	mw.logObject(self)
-	return ''
+	local table = WidgetTable{css = {['text-align'] = 'center'}, classes={'general-collapsible', 'collapsed'}}
+
+	table:addRow(self:_buildHeader())
+
+	for _, row in ipairs(self:_buildRows()) do
+		table:addRow(row)
+	end
+
+	table:setContext{self._widgetInjector}
+
+	local wrapper = mw.html.create('div')
+	for _, node in ipairs(WidgetFactory.work(table, self._widgetInjector)) do
+		wrapper:node(node)
+	end
+
+	return wrapper
+end
+
+function PrizePool:_buildHeader()
+	local headerRow = WidgetTableRow{css = {['font-weight'] = 'bold'}}
+
+	headerRow:addCell(WidgetTableCell{content = {'Place'}})
+
+	for _, prize in ipairs(self.prizes) do
+		local prizeTypeData = self.prizeTypes[prize.type]
+		local cell = prizeTypeData.headerDisplay(prize.data)
+		headerRow:addCell(cell)
+	end
+
+	-- TODO: Add support for party types
+	headerRow:addCell(WidgetTableCell{content = {'Team'}})
+
+	return headerRow
+end
+
+function PrizePool:_buildRows()
+	local rows = {}
+	for _, placement in ipairs(self.placements) do
+		-- TODO RowSpan
+		-- TODO Cutoff
+
+		for _, opponent in ipairs(placement.opponents) do
+			local row = WidgetTableRow{}
+
+			row:addClass(placement:getBackground())
+			row:addCell(WidgetTableCell{
+				content = {placement:getMedal() or '' , NON_BREAKING_SPACE, placement:displayPlace()},
+				css = {['font-weight'] = 'bolder'}
+			})
+
+			for _, prize in ipairs(self.prizes) do
+				local prizeTypeData = self.prizeTypes[prize.type]
+				local reward = opponent.prizeRewards[prize.id] or placement.prizeRewards[prize.id]
+
+				local cell
+				if reward then
+					cell = prizeTypeData.rowDisplay(prize.data, reward)
+				end
+
+				if not cell then
+					cell = WidgetTableCell{content = {DASH}}
+				end
+
+				row:addCell(cell)
+			end
+
+			-- TODO: Proper Support for Party Types
```

Comment:
might not be as easy as you think due to:
- option to have several rows within the same opponent (needed for e.g. duos where they have different teams)
- option to span it over 2 columns for the case that archon opponents are present

Reply:
> option to have several rows within the same opponent (needed for e.g. duos where they have different teams)

That would just be a new line inside a cell? ie `<br>`?

Reply:
Yeah already have that in mind